spec/models/user_category_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe UserCategory, type: :model do
+
+  let(:user) { create(:user) }
+
+  before do
+    sign_in user
+  end
+
+  it "must have user" do
+    usercategory = build(:user_category)
+    expect(usercategory).to be_valid_user
```

Comment:
nitpick: this is not correct spec. this is not checking user id is present and also we should build category in context or describe.